sbi/utils/get_nn_models.py

```diff
@@ -59,41 +48,22 @@ def classifier_nn(
         kwargs: additional custom arguments passed to downstream build functions.
     """
 
```

Comment:
Please change the above docstring: remove everything and just say that users should use `neural_nets/factor`.